capstone-project/src/main/java/com/google/sps/servlets/BlobServlet.java

```diff
@@ -89,4 +89,36 @@ private static String getBlobKey(HttpServletRequest request) {
 
         return blobKey;
     }
+
+    /** Imitates the real doPost method for testing purposes */
+    public void doGet(HttpServletRequest request, HttpServletResponse response, BlobstoreService blobstoreService) throws IOException {
```

Comment:
Can these be static? 

Also, I don't think they have to be public (default visibility is enough)